### PR TITLE
Flock Funding: Aragon Black

### PR DESCRIPTION
**Pull request by osarrouy**
This is a Finance track AGP, following our [Flock proposal](https://github.com/aragon/flock/blob/master/teams/Aragon%20Black/2019%20-%20Q3%20%26%20Q4.md).

### CODE REVIEW

File: AGPs/AGP-X.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+- 125k ANT.
+
+## Number and frequency of transfers
+
+- The \$450k payment will be divided in two chunks.
+- The 125k ANT payment is gonna be handled by the AA with respect to the process described [here](https://github.com/aragon/flock/pull/7#issuecomment-480275930) and [here](https://github.com/aragon/flock/pull/7#issuecomment-481013524).
+
+## Purpose of the transfer
+
+Aragon Flock proposal, as [stated in the Flock repo](https://github.com/aragon/flock/blob/master/teams/Aragon%20Black/2019%20-%20Q3%20%26%20Q4.md). The proposal was [pre-approved and merged](https://github.com/aragon/flock/pull/7) into the Flock repo on April 8th, 2019.
```
Comment (john-light) [inline AGPs/AGP-X.md]:
```suggestion
Aragon Flock proposal, as [stated in the Flock repo](https://github.com/aragon/flock/blob/master/teams/Aragon%20Black/2019%20-%20Q3%20%26%20Q4.md). The proposal was [pre-approved and merged](https://github.com/aragon/flock/pull/7) into the Flock repo on April 8th, 2019.

Last proposal commit: https://github.com/aragon/flock/commit/0caa233e05e1abd2816560b043527aa16d8a47a2
Proposal SHA-256 hash: b6b3aff60b86d916b9121dcfcc92acb335a780dd4e0847d782ee8d6342300c7f
```

**Comment by osarrouy** [comment]
That sounds like a good idea, indeed :) Just accepted the changes!

**Comment by john-light** [comment]
Please let the AGP editors know when your proposal has been finalized and you would like to submit it for final review by the Aragon Association. The deadline to finalize and submit your proposal is 16:00 April 11th.

**Comment by osarrouy** [comment]
Hey @john-light,

It's all good on our side!

Thanks.

**Pull request merged**